pkgs/applications/virtualization/slirp4netns/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, autoconf, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  name = "slirp4netns-${version}";
+  version = "0.3.0-alpha.2";
+
+  src = fetchFromGitHub {
+    owner = "rootless-containers";
+    repo = "slirp4netns";
+    rev = "v${version}";
+    sha256 = "163nwdwi1qigma1c5svm8llgd8pn4sbkchw67ry3v0gfxa9mxibk";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/rootless-containers/slirp4netns;
+    description = "User-mode networking for unprivileged network namespaces";
+    license = licenses.gpl2;
+    platforms = platforms.linux;
```

Comment:
Would you like to maintain this package? In this case, could you add youself in the maintainer list :)

Reply:
No, not really. I needed it for an experiment, but I probably won't continue using it.